src/components/CommentList/CommentList.jsx

```diff
@@ -1 +1,12 @@
-export const CommentList = () => <>Put the list here</>;
+import { CommentInfo } from '../CommentInfo/CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => {
+  return (
+    <div className="CommentList">
+      {comments.map(comment => (
+        <CommentInfo key={comment.id} comment={comment} />
+      ))}
+    </div>
+  );
+};
```

Comment:
if there is only return you can just write `export const CommentList = ({ comments }) => ( ... );`